modules/frontend/combiner/metrics_query_range.go

```diff
@@ -16,11 +17,13 @@ var _ GRPCCombiner[*tempopb.QueryRangeResponse] = (*genericCombiner[*tempopb.Que
 // NewQueryRange returns a query range combiner.
 func NewQueryRange(req *tempopb.QueryRangeRequest) (Combiner, error) {
 	combiner, err := traceql.QueryRangeCombinerFor(req, traceql.AggregateModeFinal)
+	maxSeries := int(req.MaxSeries)
```

Comment:
are we handling Status = Partial being returned from the queriers here? Shouldn't we pass that back to the caller even if `combiner.MaxSeriesReached() == false` here?

Reply:
the combiner will set MaxSeriesReached() to true if Status = Partial is returned from the querier